kitsune/wiki/views.py

```diff
@@ -1453,3 +1479,116 @@ def what_links_here(request, document_slug):
     }
 
     return render(request, 'wiki/what_links_here.html', c)
+
+
+def get_fallback_locale(doc, request):
+    """Get best fallback local based on locale Mapping"""
+
+    # Get localized article versions with current revisions
+    translated_locales = (doc.translations.all()
+                                          .values_list('locale', flat=True)
+                                          .exclude(current_revision=None))
+    # Get the ACCEPT_LANGUAGE header from the user's request
+    browser_language_code = request.META.get('HTTP_ACCEPT_LANGUAGE', None)
```

Comment:
The comment here only restates the code. It doesn't add anything.

`None` is the default for `.get()`. You can leave it off.